apis/openstack-swift/src/main/java/org/jclouds/openstack/swift/v1/blobstore/RegionScopedSwiftBlobStore.java

```diff
@@ -819,10 +817,7 @@ public void run() {
                }
             }
             // Finished writing results to stream
-            try {
-               output.close();
-            } catch (IOException e) {
-            }
+            Closeables2.closeQuietly(output);
```

Comment:
Also, would it make sense to enclose the entire `for` block in a try/finally block, to make sur everything is always closed, without having to do it explicitly in the middle of the code and concrete catch blocks?